test/verify/check-storage-anaconda

```diff
@@ -549,6 +544,86 @@ class TestStorageAnaconda(storagelib.StorageCase):
         b.click(self.card_desc("MDRAID disk", "MDRAID device") + " button")
         b.wait_in_text("body", "Not found")
 
+    def testNonEmpty(self):
+        b = self.browser
+        m = self.machine
+
+        disk = self.add_loopback_disk(name="loop10")
+
+        anaconda_config = {
+            "mount_point_prefix": "/sysroot",
+            "available_devices": [disk],
+        }
+
+        # Create some content on the disk
+        m.execute(f"echo einszweidrei | cryptsetup luksFormat --pbkdf-memory 32768 {disk}")
+        m.execute(f"echo einszweidrei | cryptsetup luksOpen {disk} dm-test")
+        m.execute("mkfs.ext4 /dev/mapper/dm-test; mount /dev/mapper/dm-test /mnt; echo Hi >/mnt/hello; umount /mnt")
+        m.execute("cryptsetup close dm-test")
```

Comment:
A test for a Filesystem with only directories would be nice as well.

Reply:
Yes, agreed.

Reply:
Done.